### PR TITLE
M2 HAT+ doc

### PR DESCRIPTION
**Pull request by nathan-contino**

### CODE REVIEW

File: documentation/asciidoc/accessories/m2-hat-plus/about.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+. Reboot your Raspberry Pi with `sudo reboot`.
+
+For more information, see xref:../computers/raspberry-pi.adoc#nvme-ssd-boot[NVMe boot].
+
+== Schematics
+
+.Schematics for the Raspberry Pi M.2 HAT+
+image::images/m2-hat-plus-schematics.png[width="80%"]
+
+Schematics are also available as a https://datasheets.raspberrypi.com/m2-hat-plus/raspberry-pi-m2-hat-plus-schematics[PDF].
```
Comment (nathan-contino) [inline documentation/asciidoc/accessories/m2-hat-plus/about.adoc:124]:
```suggestion
Schematics are also available as a https://datasheets.raspberrypi.com/m2-hat-plus/raspberry-pi-m2-hat-plus-schematics.pdf[PDF].
```

File: documentation/asciidoc/accessories/m2-hat-plus/about.adoc [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+----
++
+If you see December 6, 2023 or a later date, proceed to the next step. If you see a date earlier than December 6, 2023, run the following command to open the Raspberry Pi Configuration CLI:
++
+[source,console]
+----
+$ sudo raspi-config
+----
++
+Under `Advanced Options` > `Bootloader Version`, choose `Latest`.
```
Comment (lurch) [inline documentation/asciidoc/accessories/m2-hat-plus/about.adoc:56]: ...and then select "Finish"
?

File: documentation/asciidoc/accessories/m2-hat-plus/about.adoc [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
++
+--
+image::images/installation-07.png[width="60%"]
+--
+
+WARNING: Always disconnect your Raspberry Pi from power before connecting or disconnecting a device from the M.2 slot.
+
+== Boot from NVMe
+
+To boot from an NVMe drive attached to the M.2 HAT+, complete the following steps:
```
Comment (lurch) [inline documentation/asciidoc/accessories/m2-hat-plus/about.adoc:110]: Perhaps this should also mention that you need to copy/install an OS to the NVMe drive?

File: documentation/asciidoc/computers/raspberry-pi/boot-nvme.adoc [hunk 4]
```diff
@@ -1,35 +1,76 @@
...
 
-=== Required hardware
+=== Prerequisites
 
-You need an NVMe M.2 SSD. You cannot plug an M.2 SSD directly into the PCIe slot on the IO board; an adapter is needed. Be careful to get the correct type: a suitable adapter can be found online by searching for "PCI-E 3.0 ×1 lane to M.2 NGFF M-Key SSD NVMe PCI Express adapter card".
+==== Hardware
 
-The latest version of Raspberry Pi OS supports booting from NVMe drives. To check that your NVMe drive is connected correctly, boot Raspberry Pi OS from another drive and run `ls -l /dev/nvme*`. Example output is shown below.
+* NVMe M.2 SSD
+* an adapter to convert from PCIe FFC to an M.2 standard.
```
Comment (lurch) [inline documentation/asciidoc/computers/raspberry-pi/boot-nvme.adoc:10]: Minor nitpick, but whilst the Pi5 has a PCIe FFC, the CM4IO board has a regular PCIe slot.

**Pull request merged**